notebooks/retriever-reader-pipelines/prompt_reader_with_GPT.ipynb

```diff
@@ -0,0 +1,254 @@
+{
```

Comment:
Is this notebook only about using GPT based reader w/o the retriever ? Shouldn't we move it to /notebooks/readers/ then ?

Reply:
We want this notebook because it shows a comparison of Instruct GPT output:   not using retrieval Instruct GPT  gets the wrong answers.  Using retrieval, Instruct GPT gets the correct answers. If you notice the same question is asked 2 times, once with retrieval and once without. This might come in handy and still uses a QA pipeline.